### PR TITLE
Rebrand account card

### PR DESCRIPTION
**Pull request by etharner**

### CODE REVIEW

File: src/icons/account-card/collapse.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```
Comment (CedrikNikita) [inline src/icons/account-card/collapse.svg:1]:
```suggestion
<svg viewBox="0 0 24 24" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

File: src/icons/account-card/expand.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```
Comment (CedrikNikita) [inline src/icons/account-card/expand.svg:1]:
```suggestion
<svg viewBox="0 0 24 24" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```
```suggestion
<svg width="24" height="24" viewBox="0 0 24 24" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

File: src/popup/router/components/AccountCard.vue [hunk 3]
```diff
@@ -1,45 +1,37 @@
 <template>
   <div
-    :class="['account-card', { subaccount: idx !== 0, 'first-subaccount': idx === 1 }]"
+    :class="[
+      'account-card',
+      { subaccount: idx !== 0, 'first-subaccount': idx === 1 }]"
```
Comment (CedrikNikita) [inline src/popup/router/components/AccountCard.vue:5]: Why do we need this changes?

File: src/popup/router/components/AccountSwitcher.vue [hunk 4]
```diff
@@ -68,6 +66,7 @@ export default {
   flex-direction: column;
   border-radius: 0 0 10px 10px;
   padding-top: 32px;
+  margin-bottom: 24px;
```
Comment (CedrikNikita) [inline src/popup/router/components/AccountSwitcher.vue:69]: margin with `.buttons` should be 12px (clarify in designs),
without should be 16px

File: src/popup/router/components/AccountCard.vue [hunk 5]
```diff
@@ -1,45 +1,37 @@
...
     :style="cssVar"
   >
     <AccountInfo v-bind="$attrs" />
-    <BalanceInfo v-bind="$attrs" />
-    <div class="arrows">
```
Comment (CedrikNikita) [inline src/popup/router/components/AccountCard.vue:8]: class="arrow" related css should be also removed
https://github.com/aeternity/superhero-wallet/blob/44939e695dd12c46738e1e5c1a99359bb4163849/src/popup/router/components/AccountCard.vue#L95-L118

File: src/popup/router/components/AccountInfo.vue [hunk 6]
```diff
@@ -1,32 +1,39 @@
...
+      <div>
+        <ButtonPlain
+          v-clipboard:copy="accounts[idx].address"
+          data-cy="copy"
+          @click="copy"
+        >
+          <Copy />
+        </ButtonPlain>
+        <ButtonPlain
+          v-if="UNFINISHED_FEATURES && idx === 0"
```
Comment (CedrikNikita) [inline src/popup/router/components/AccountInfo.vue:18]:
```suggestion
          v-if="UNFINISHED_FEATURES"
```
Seems that `createAccount` button should be visible in each card. Can you elaborate on this @paolomolo?

File: src/popup/router/components/AccountInfo.vue [hunk 7]
```diff
@@ -1,32 +1,39 @@
...
+          @click="createAccount"
+        >
+          <Add />
+        </ButtonPlain>
+        <ButtonPlain
+          v-if="idx !== 0 && $route.path === '/accounts'"
```
Comment (CedrikNikita) [inline src/popup/router/components/AccountInfo.vue:24]: Account with idx=0 can be deleted now aswell, there should be different wording for this case aswell

**Comment by CedrikNikita** [comment]
AccountCard also is 16px smaller, than it should be

**Comment by paolomolo** [comment]
Feedback for the account cards, in general please update margins and spacings as in the prototype. Thanks!

**account card**:
- width: 328px
- margin to left and right screen edge: 16px
- margin-top to navbar: 12px
- margin to green account switcher tabs: 12px
- margin to next card: 4px
- remove downward animation on slide into view
- add hotspot on next card to slide it into view
- add drag navigation to navigate
- please review spacing for account card settings screen
- improve collaps animation:
  - add `overflow: hidden` to account card to hide balance with animation

---

**account switcher tabs**:
- height: 10px (without border)
- bottom margin: 12px

---
 
**account card action buttons** (copy, plus, settings):
- spacing between: 8px (remove whitespace)

**Comment by etharner** [comment]
> Feedback for the account cards, in general please update margins and spacings as in the prototype. Thanks!
> 
@paolomolo I've fixed everything beside user gestures, I think it should be done separately to separate stylistic and algorithmic changes

**Review by CedrikNikita — APPROVED** [review]
Since this changes only style/markup related, the rest of logic changes could be made in other PR.

File: src/popup/App.vue [hunk 8]
```diff
@@ -203,6 +203,11 @@ body {
         padding-left: 0;
         padding-right: 0;
       }
+
+      &:not(.transactions) ::v-deep .account-switcher {
+        margin-left: -20px;
+        margin-right: -20px;
+      }
```
Comment (CedrikNikita) [inline src/popup/App.vue:210]: I did not undestand the purpose of this code

**Review by CedrikNikita — CHANGES_REQUESTED** [review]

**Pull request merged**